cgp/ea/mu_plus_lambda.py

```diff
@@ -52,6 +52,8 @@ def __init__(
         self.local_search = local_search
         self.k_local_search = k_local_search
 
+        self.objective_calls = 0
```

Comment:
```suggestion
        self.n_objective_calls = 0
```
to indicate that this is a number. we use it in a couple of places, so why not here as well as it can increase readability.